src/main/java/pl/sda/rentacar/domain/employee/EmployeeService.java

```diff
@@ -0,0 +1,50 @@
+package pl.sda.rentacar.domain.employee;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static pl.sda.rentacar.domain.employee.EmployeeMapper.MAPPER;
+
+@Service
+@RequiredArgsConstructor
+public class EmployeeService {
```

Comment:
This class is not covered with tests. Please cover it with integration tests - and only then, do the refactor.